src/Commands/LivewireMakeCommandParser.php

```diff
@@ -52,8 +57,10 @@ public function classFile()
 
     public function classNamespace()
     {
+        $namespace = config('livewire.namespace', $this->getAppNamespace());
```

Comment:
I'd maybe group all the items fetched from the config in the constructor.

Reply:
So funny I had it that way and then moved it into the method.